mrcnn/scripts/train.py

```diff
@@ -0,0 +1,190 @@
+"""
+Training module for maskrcnn on Bottles, Boxes and Candy bags.
+"""
+
+from config import *
+import h5py
+from imgaug import augmenters as iaa
+from sklearn.model_selection import train_test_split
+from random import randint
+import numpy as np
+import json
+
+
+class ClomaskDataset(utils.Dataset):
+    """
+        Class wrapper for clomask dataset.
+    """
+
+    def load_shapes(self, id_list, train_path):
+        """
+           Initialize the class with dataset info.
+        """
+        with open(ROOT_DIR + 'id_map.json') as mapping_file:
```

Comment:
Could you check in the `id_map.json` file? Or I can add that long list of class labels in `config.py` (with "book" becomes "bags").

Reply:
@lmtoan I have added the ```id_map.json``` to the pull request.